### PR TITLE
Business and interview pages

### PR DESCRIPTION
**Pull request by haakonbu**
Pages with info regarding companies, interviews with former students, statistics etc.

This branch will implement the app with a React frontend.

### CODE REVIEW

File: chemie/corporate/templates/corporate/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          fortelle om sin virksomhet, hvilke arbeidsoppgaver som venter en nyutdannet sivilingeniør i bedriften og andre
+          emner som kan interessere studentene.
+        </p>
+        <button class="btn btn-primary">Bedrifter</button>
+        <button class="btn btn-secondary">Intervjuer</button>
+      </div>
+    </div>
+    <div class="col-sm-12 col-md-3">
+      <div class="card">
+        <img src="../../../chemie/static/images/indkom.jpg" class="card-img-top" alt="...">
```
Comment (hambro) [inline chemie/corporate/templates/corporate/index.html]: Bedre å bruke `{% static ... %}`  enn relativ src.

File: chemie/corporate/urls.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+from django.urls import include, path
+from . import views
+
+app_name = "corporate"
+
+urlpatterns = [
+    url(r"^$", views.index, name="index"),
```
Comment (hambro) [inline chemie/corporate/urls.py]: Kan bruke `path` på denne også. 
Trenge vel ikke trailing slash på de 2 URLene under.

File: chemie/corporate/templates/corporate/list_companies.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+</head>
+<body>
+
+{% if companies %}
+
+<ul>
+    {% for company in companies %}
+    <li>{{ company.name }}</li>
+    {% endfor %}
+</ul>
```
Comment (hambro) [inline chemie/corporate/templates/corporate/list_companies.html]: Dette "if companies ... for company ... Kan egentlig byttes ut med [for … empty](https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#for-empty)

**Review by hambro — DISMISSED** [review]

File: chemie/corporate/views.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+from .models import Interview, JobAdvertisement
+
+from chemie.committees.models import Committee
+from chemie.events.models import Bedpres, Social
+from .forms import CreateInterviewForm, CreateJobForm
+
+
+def index(request):
+    indkom = Committee.objects.get(title="Industrikomiteen")
```
Comment (Geirhov) [inline chemie/corporate/views.py:14]: This would cause an error if the name of the committee is ever changed.

**Review by Geirhov — CHANGES_REQUESTED** [review]
Overall it looks really great, and this is a great addition to the website. There are however a few things that need to be addressed before it is ready for merging. 

First off all the HC-image on the index page is covering the title, i'm suspecting that this error is caused by our screensizes being different, but it looks a bit unprofessional.  
Furthermore i discovered a bug where when i try to delete an interview, one of the job announcements is deleted instead. This needs to be fixed. 
Finally i am missing the ability to sort by specialization when lisiting jobs, but this is not necessary to get the app live and running.

**Review by Geirhov — APPROVED** [review]
This looks really good, excellent work!
I am missing the ability to sort the jobs by specialization, but that's not essential for the functionality. 
I'm really liking the look of the app, and it's gonna be exciting to see how the look of the website changes in the coming months.

File: chemie/corporate/views.py (hunk 4 above)
Comment (bendikstoasannes) [reply chemie/corporate/views.py:14]: Let's unngå that

**Pull request merged**